docs/en/operation/StressBench.md

```diff
@@ -408,8 +460,15 @@ For the `RegisterWorkerStream` RPC.
 
 This benchmark calls the RPC only once.
 
-`RegisterWorkerStream` is similar to `RegisterWorker`, except that this RPC allows streaming of 
-the huge block list. 
+Simulates concurrent worker registration using the streaming RPC implementation (default since 
```

Comment:
I added a doc link too, refer to this
https://docs.alluxio.io/os/user/stable/en/operation/Scalability-Tuning.html#streaming-worker-registration